src/main/java/gregtech/common/metatileentities/multi/MetaTileEntityLargeBoiler.java

```diff
@@ -126,7 +126,7 @@ public boolean isActive() {
 
     @Override
     protected BlockPattern createStructurePattern() {
-        return boilerType == null ? null : FactoryBlockPattern.start()
+        return FactoryBlockPattern.start()
```

Comment:
Instead of now NPEing here, should BoilerType be set to a default type if null?

Reply:
Boiler type is expected to be present. If we want to support custom boiler types we should look into expanding current implementation, instead of allowing nulls.